mock-core-configs/etc/mock/templates/fedora-branched.tpl

```diff
@@ -29,6 +29,7 @@ metadata_expire=0
 best=1
 module_platform_id=platform:f{{ releasever }}
 protected_packages=
+user_agent=mock (Fedora {{ releasever }}; mock; {{ target_arch }})
```

Comment:
Don't you want to use the standard formatt `Mock/<Version> <comment>`?  `tito.props` describes `VERSION_TEMPLATE` that could help.

Also, I'd slightly prefer using something from `python-distro` instead of `target_arch`.  The target arch should be obvious from the downloaded package file name (path).

Reply:
That is a question. I even considered just "mock" as the version or arch will not be too much helpful.

Reply:
Ok, per RFC 7231, product version is [optional](https://tools.ietf.org/html/rfc7231#page-46).  Previously I looked at some wrong interpretation by Mozilla.

Reply:
Otherwise, I don't know.  For us (as mock maintainers) is interesting to hear
where people run mock tool (Fedora version and architercture, e.g.).

The `target_arch` seems to be redundant info.

So good enough format would be e.g.:
`Mock/2.4 (Fedora 32; x86_64)`